src/services/itemMembership/index.ts

```diff
@@ -36,16 +38,14 @@ const plugin: FastifyPluginAsync = async (fastify) => {
         fastify.register(fastifyCors, fastify.corsPluginOptions);
       }
 
-      // if (WEBSOCKETS_PLUGIN) {
-      //   registerItemMembershipWsHooks(
-      //     websockets,
-      //     runner,
-      //     itemsDbService,
-      //     dbService,
-      //     taskManager,
-      //     db.pool,
-      //   );
-      // }
+      if (WEBSOCKETS_PLUGIN) {
```

Comment:
I think you can remove the `WEBSOCKETS_PLUGIN` check, we don't use it anymore for the items